benchmark/wildtime_benchmarks/benchmark_utils.py

```diff
@@ -57,7 +57,7 @@ def setup_logger():
 
 
 def create_fake_timestamp(year: int, base_year: int) -> int:
-    timestamp = ((year - base_year) * DAY_LENGTH_SECONDS) + 1
+    timestamp = ((year - base_year) * DAY_LENGTH_SECONDS)
```

Comment:
not sure why previously there is a `+1`. Without it every timestamp is neater.

Reply:
Hmm. IIRC, it has something to do with the trigger. It might have been some reasoning like "we need a day + 1 so that the trigger fires" but I think that is actually not true. If this works for the time trigger, then we can get rid of the + 1 :)

Reply:
Yeah it works for my past experiments. Also @jenny011 confirmed as well without `+1` everything is easier to manage.